### DISCUSSION TITLE
Games that use Battle Eye anticheat system trigger

### DISCUSSION THREAD

**Discussion by 0x01001100** [valinet/ExplorerPatcher#3712] [General]
Some games that use BattleEye anticheat are triggered by  --triestoblock 
 "C:\Program Files\ExplorerPatcher\ExplorerPatcher.IA-32.dll". 
 and take measures, depending on each game, experienced some....
 using valinetExplorerPatcher 22621.3880.66.6, maybe has to do with the fact now binbowsOS is really being triggered even more now by this "solution" from valinet..

 ¿Any though on this?..

**Comment by Amrsatrio** [comment]
Disable shell extensions in Properties > File Explorer.
